students/DanCornutt/session03/mailroom.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""
+mailroom assignment
+"""
+from operator import itemgetter
+
+donors = [("Fred Jones", [100.01, 200, 300]),
+    ("Amy Shumer", [2000, 4000, 1000]),
+    ("Billy Bills", [1020, 20440.55, 300]),
+    ("Bob Sherlock", [10, 20, 30]),
+    ("Tom Johnson", [100, 200, 900])]
+
+
+def thank_you():
+    print('do the thank you thing now!')
+    answer = ""
+    while answer != "Q":
+        print("Please input a full name of donor or type 'list' if you would like to see the entire list")
+        answer = input("=> ")
+        answer = answer.strip()
+        answer = answer.title()
+        if answer == 'List':
```

Comment:
I like to use just the first letter, so typos don't mess people up -- but that's minor.